test/reactive-handler.spec.ts

```diff
@@ -594,7 +594,7 @@ describe('ReactiveHandler', () => {
         expect(accessSpy).toHaveBeenLastCalledWith(obj.foo, 'bar');
     });
 
-    describe.skip('issue#20 - getOwnPropertyDescriptor', () => {
+    describe('issue#20 - getOwnPropertyDescriptor', () => {
```

Comment:
We should add `valueObserved` and `valueMutated` traps for all the tests to ensure that the membrane is catching those mutations properly.

Reply:
This is actually very interesting. `valueObserved` is marked when the descriptor is accessed, and `valueMutated` can't really be used effectively.